acceptance_tests/features/pages/survey.py

```diff
@@ -27,3 +27,7 @@ def get_surveys():
 def click_qbs_survey_link():
     link = browser.find_by_name('survey-link-QBS')
     link.click()
+
+
+def click_edit_survey_details_button():
+    browser.click_link_by_id('edit-survey-details-199')
```

Comment:
In general I don't think we should be changing the details for existing surveys. We should be creating a new survey for the test then editing that to prevent us contaminating the data for other tests.
We'd need the create survey endpoints to be present for this so might be good to update in the future

Reply:
@andrewmil Yeah I talked about that with Tej this morning currently these 2 tests:

```Failing scenarios:
  acceptance_tests/features/view_collection_exercises_data.feature:42  Ensure collection exercise exists for a survey -- @1.13 
  acceptance_tests/features/view_survey_data.feature:9  View all default surveys```

fail because they are asserting for survey data that has been changed and we only do the reset before the acceptance tests startup. So I can put a temporary fix in to update these values back to what they were before after the tests for updating survey details have passed.